Makefile

```diff
@@ -9,7 +9,7 @@ VERSION ?= "0.0.0-dev-$(shell git rev-parse --short HEAD)"
 # since we're using github.com/grpc-ecosystem/grpc-gateway it needs to be manually cloned and put somewhere
 # close to the project and this env var should point to "third_party/googleapis" subdir.
 # IMPORTANT - check currently used version of github.com/grpc-ecosystem/grpc-gateway to use exactly the same tag
-GOOGLEAPIS_DIR ?= "../googleapis"
+GOOGLEAPIS_DIR ?= "../grpc-gateway/third_party/googleapis"
```

Comment:
what do you think if we use submodules to clone third parties?
this way we ensure `googleapis` is at correct version

Reply:
git submodules is the worst thing I've ever used for vendoring deps, I'll think how to do this in an easier way